Source/DafnyLanguageServer/Handlers/Custom/VerificationHandler.cs

```diff
@@ -43,12 +43,10 @@ public class VerificationHandler : IJsonRpcRequestHandler<VerificationParams, bo
     var translatedDocument = await documentEntry.TranslatedDocument;
     var requestPosition = request.Position;
     var tasks = GetTasksAtPosition(translatedDocument, requestPosition).ToList();
-    foreach (var taskToRun in tasks) {
-      var verifiedDocuments = documentLoader.Verify(translatedDocument, taskToRun, CancellationToken.None);
-      documentEntry.Observe(verifiedDocuments);
```

Comment:
Why do we not need to observe the task any more?

Reply:
Because now there's a single VerificationUpdates Observable per document that's already subscribed to by the publisher.

The problem was that `documentEntry.Observe` kept the observations from calls to it in order, so then if you verify task1 and then task2, but task2 completes before task1, you'll still see the notifications of task1 before task2.